### PR TITLE
FDG-5442: Update Hero Section Styling and Spacing

### PR DESCRIPTION
**Pull request by j-hertzler**
https://federal-spending-transparency.atlassian.net/browse/FDG-5442

No updates to code coverage as this ticket only focused on styling

### CODE REVIEW

File: src/layouts/explainer/hero-image/hero-image.tsx [hunk 1]
```diff
@@ -16,10 +16,15 @@ const HeroImage: FunctionComponent<IHeroImage> = ({
   return (
     <div
       className={mainContainer}
-      style={{ backgroundColor: primaryColor }}
+      style={{ borderBottomColor:primaryColor }}
```
Comment (wyattch) [inline src/layouts/explainer/hero-image/hero-image.tsx:19]: tiny tiny nitpick here, but I think there should be a space here after the semicolon

**Review by wyattch — CHANGES_REQUESTED** [review]
Looks like the hyperlink in the hero section needs to have it's color updated to match design mocks

**Comment by j-hertzler** [comment]
> Looks like the hyperlink in the hero section needs to have it's color updated to match design mocks

@wyattch The ticket specified that the hyperlink coloring on the mock was not up to date

**Review by wyattch — APPROVED** [review]
Everything looks good! Approved

**Pull request merged**